artifactory/utils/dependenciesutils.go

```diff
@@ -22,6 +22,7 @@ const (
 	// configured to proxy releases.jfrog.io.
 	// This env var should store a server ID and a remote repository in form of '<ServerID>/<RemoteRepo>'
 	ExtractorsRemoteEnv = "JFROG_CLI_EXTRACTORS_REMOTE"
```

Comment:
Let's add to the beginning of the description for this variable that it is deprecated and replaced with `releasesRemoteEnv`.

Reply:
done